headless/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM fedora:latest AS builder
+
+ARG TARGETARCH
+
+RUN yum install gcc glibc-devel zlib-devel libstdc++-static curl git -y
+RUN yum install zlib-devel zlib-static -y
+RUN yum install glibc-static -y
+
+# Install Enterprise GraalVM and Dependencies (needed to compile native image with JNI invocations)
+RUN cd / && mkdir graal
+
+COPY resources/graalvm-ee-java19-${TARGETARCH}.tar.gz /graal/graal.tar
+COPY resources/graal-config /graal/config
+
+WORKDIR /graal
+
+RUN tar -xzf graal.tar && rm graal.tar
+
+ENV PATH="${PATH}:/graal/graalvm-ee-java19-22.3.0/bin"
```

Comment:
If GraalVM is downloaded (instead of `COPY`ed from local dir) when building this image, should we do something with this `22.3.0` version number? I mean, it might be changed later if we're download the "latest" version of GraalVM.